nautobot/docs/administration/nautobot-server.md

```diff
@@ -383,3 +383,13 @@ Listening on port http://localhost:9000. Stop with CONTROL-C.
 ```
 
 Please see the guide on [Troubleshooting Webhooks](../models/extras/webhook.md#troubleshooting) for more information.
+
+### `startplugin`
+
+`nautobot-server startplugin [app_name]`
```

Comment:
Looks like this file uses `[]` interchangeably to denote both optional arguments and variable arguments. Personally I'd rather be clearer here:

```suggestion
`nautobot-server startplugin <plugin_name> [<directory>]`
```